Makefile

```diff
@@ -5,9 +5,17 @@ default: Dockerfile
 	docker build -t infoblox/helm:$(IMAGE_VERSION) .
 	docker push infoblox/helm:$(IMAGE_VERSION)
 
+build: Dockerfile
```

Comment:
[DRY](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself) concern - please remove recipe for the `default` target and change to `default: build docker-push`